server/src/test/java/org/eclipse/lsp/cobol/service/delegates/completions/SnippetCompletionTest.java

```diff
@@ -35,12 +35,12 @@
   private static final String DOCUMENTATION_TEXT = "WRITE item";
   private static final String LABEL = "WRITE";
 
-  private SnippetCompletion completion = new SnippetCompletion(new Snippets());
+  private final SnippetCompletion completion = new SnippetCompletion(new Snippets());
 
   @Test
   void testCompletionEmptyResult() {
     assertThat(
-        completion.getCompletionItems(
+            new SnippetCompletion(new Snippets()).getCompletionItems(
```

Comment:
Why are you creating a new one?

Reply:
It is just to avoid shared state for the test.